script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt

```diff
@@ -266,62 +275,70 @@ Right now, I'm the only one who will pay the price for my own foolishness.
 [sha:9ce15b78a2b0d7f15dd06aad390fe59b861f58c6]{
 -- Page 241, Offset 40137.
 -- 全身を縛る硬直から解放されないまま、俺は一枚の木の葉のように、空中へと投げ出された。
-Without being released from the stiffening of my entire body, I am flung into the air like a single tree leaf.
+I'm sent flying into the air like a leaf on the wind, my body still rigidly locked in place.
 }
 [sha:9d4055bad51061c4e5ed3d8f8b32b81d11b300a3]{
 -- Page 242, Offset 40138.
 -- ……白い気流の中、無様に落ちていく。
-...I fall awkwardly into the middle of the white air currents.
+I tumble down head over heels, buffeted by the white air currents.
 }
 [sha:86e3a8df03260999f42da435406dccc35a37ab39]{
 -- Page 242, Offset 40139.
 -- 自分が何処にいるのか、落下に備えて何をするべきかも、考えられない。
-I can't think about where I am, or even about how I should prepare for the fall.
+I don't even wonder where I am,
+ or what I should do to try and soften my landing.
 }
 [sha:9119f355eaf3f228b45135d2953a8d9c5c8e8b3e]{
 -- Page 243, Offset 40140.
 -- ……当然だ。自覚が無かっただけで、何もかも限界だった。
-...That's natural. I've simply lost all self-awareness, so I've practically reached my limit.
+It's only natural...
+ I hadn't noticed, but I was well past my limits.
 }
 [sha:711a80bda6e7df935cff010eb72559a1980ff13e]{
 -- Page 243, Offset 40141.
 -- 昨日まで普通に走るだけで息を切らしていたクセに、100メートルを５秒で走るだの、ビルの上を駆け回るだの、はては巨人に跳び移るだの。
-Despite being someone who, from as early as yesterday, would be out of breath from simply running, I ran 100 meters in just 5 second, jumped around on top of buildings, and on top of that, soared my way over to that giant.
+Despite being someone who as of yesterday would be out of breath after a light jog,
+ I just ran a hundred meters in five seconds,
+ dashed around on top of buildings,
+ and jumped all the way over to that giant.
 }
 [sha:f458aa8665eb2b31b1d1c30520ab04de67a9fbed]{
 -- Page 244, Offset 40142.
 -- 全身の筋肉は幾度となく断裂し、その度につなぎ合わされていた。通常の何百倍もの速さの新陳代謝。本来その回復に使われるだけの“時間”を、俺の脳は消費している。
-Every muscle in my body has been torn on multiple occasions, and they were pieced back together each and every time. My metabolism is 100 times faster than normal. The 'time' that would normally be used for recovery is consumed by my brain instead.
+Every muscle in my body has been torn and stitched back together countless times.
+ My metabolism is running hundreds of times faster than normal.
+ My brain is zeroing out the 'time' that would otherwise be needed to heal those wounds.
```

Comment:
The last sentence is about vampire shit, and just do it a bit more literally. "Yeah but that's nonsense". It's fine. It's Nasu's nonsense.
>...been torn asunder and then stitched...
>...
>The 'time' that would ordinarily be used whenever I performed such healing is instead consumed by my brain.